src/Microsoft.TestPlatform.Extensions.BlameDataCollector/Constants.cs

```diff
@@ -122,6 +122,11 @@ internal static class Constants
     /// </summary>
     public const string MiniConfigurationValue = "Mini";
 
+    /// <summary>
+    /// Configuration value for none
+    /// </summary>
+    public const string NoneConfigurationValue = "None";
```

Comment:
Is it used?

Reply:
Nope, I changed the whole logic around parsing params to use enums and removed it.